src/lib/arrays.js

```diff
@@ -0,0 +1,67 @@
+exports.getNthElement = (index, array) => {
+  return array[index % array.length]; // remainder of index/array length equals given parameter in questions
+};
+
+exports.arrayToCSVString = array => {
```

Comment:
if you're returning one line you can write it like:

array => array.toString();